pkgs/development/python-modules/deezer-python/default.nix

```
@@ -0,0 +1,22 @@
+{ lib, buildPythonPackage, fetchPypi, isPy3k, requests, tornado }:
+
+buildPythonPackage rec {
+  pname = "deezer-python";
+  version = "2.2.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0j3p0ckvkxaq3ng2f13bjhyxxiz0acm7yrsn7x5hmyjy0sk3mjcm";
+  };
+
+  disabled = !isPy3k;
+
+  propagatedBuildInputs = [ requests tornado ];
+
```

```suggestion
  propagatedBuildInputs = [ requests tornado ];

  pythonImportsCheck = [ "deezer" ];
```

---

Also one input per line.
```suggestion
  propagatedBuildInputs = [
    requests
    tornado
  ];
```